examples/controller_avian.rs

```diff
@@ -0,0 +1,44 @@
+//! This example shows a simple player-controlled object using Avian2D physics. You can move the object using arrow keys.
+
+use avian2d::prelude::*;
+use bevy::prelude::*;
+use bevy_ecs_tiled::prelude::*;
+use bevy_ecs_tilemap::prelude::*;
+
+mod helper;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_plugins(TilemapPlugin)
+        .add_plugins(TiledMapPlugin)
+        .add_plugins(helper::HelperPlugin)
+        .add_plugins(PhysicsPlugins::default().with_length_unit(100.0))
+        .add_plugins(PhysicsDebugPlugin::default())
+        .add_systems(Startup, startup)
+        .run();
+}
+
+fn startup(mut commands: Commands, asset_server: Res<AssetServer>) {
+    commands.spawn(Camera2dBundle::default());
+
+    let map_handle: Handle<TiledMap> = asset_server.load("multiple_layers_with_colliders.tmx");
+    commands.spawn(TiledMapBundle {
+        tiled_map: map_handle,
+        tiled_settings: TiledMapSettings {
+            // This is the default, but we're setting it explicitly here for clarity.
+            collision_object_names: ObjectNames::All,
+            // By default, colliders are added without associated RigidBody
+            // for this example, let's add a RigidBody::Kinematic
+            // you can also add any physic related component using this mecanism
+            collider_callback: |entity_commands| {
```

Comment:
I'm not exactly sure of this: I had to add a Kinematic Rigid Body for the Rapier example to behave correctly but it may not be needed for Avian.
If you can think of a better way to handle a player character, please do!

Reply:
The rigid body is needed in avian as well. A minor change is to use a Static rigid body for the map colliders as it makes more sense they are static bodies.